meta-mender-tegra/recipes-core/images/tegra-minimal-initramfs.bbappend

```diff
@@ -0,0 +1,5 @@
+# Prevents circular dependency on u-boot due to EXTRA_IMAGEDEPENDS.
+# See https://hub.mender.io/t/warrior-tegra-build-cicular-dependencies/757/5 and comments from Mirza
+EXTRA_IMAGEDEPENDS_remove = "u-boot"
+# Don't need mender to work with this image
+IMAGE_FSTYPES_remove = "mender mender.bmap"
```

Comment:
I think this specific problem has been fixed with this commit,

https://github.com/mendersoftware/meta-mender/commit/a944df86e0aa7e61385ea1867847f65b59f8aeb3

Reply:
Agreed, removed